site/content/entry/yuzu-progress-report-apr-2020/index.md

```diff
@@ -0,0 +1,214 @@
+The border color of textures in this game are in sRGB format, and we had some small precision errors. 
+Now you can enjoy those intensive matches!
+
+{{< single-title-imgs
+    "WAA! (Mario Tennis Aces)"
+    "./09.png"
+    "./10.png"
+  >}}
+
+[Rodrigo](https://github.com/ReinUsesLisp) added indexed QUADS to Vulkan, this brings mayor graphical fixes to `Xenoblade Chronicles 2` and `Fast RMX`! Some barrier fixes are still needed in `Xenoblade Chronicles 2` to get proper rendering, work on these rendering fixes is on-going.
+
+{{< single-title-imgs
+    "From abstract to RPG. (Xenoblade Chronicles 2)"
+    "./23.png"
+    "./24.png"
+  >}}
+
+[Blinkhawk](https://github.com/FernandoS27) implemented some optimizations to the GPU command list, improving performance 
+in Diablo 3 and many other games. 
+He also ported a Ryujinx hack made by gdkchan, `Fast GPU`, this helps force the maximum resolution in games that can dynamically change it when they run slower than their target framerate, some examples include `ARMS`, `Super Mario Odyssey`, `Splatoon 2`, `Luigi’s Mansion 3` and `Mario Tennis Aces`. 
+This is not only a graphical improvement, but also a way to avoid very high memory use on such dynamic resolution games. 
+We still recommend the use of resolution mods, but if such mods are not available for your game, Fast GPU will help you. 
+A good example of this is The `Legend of Zelda: Breath of the Wild` with the latest 1.6.0 update.
+
+{{< single-title-imgs
+    "Milk baths are over Link, go save Zelda or something. (Legend of Zelda: Breath of the Wild)"
+    "./35.png"
+    "./36.png"
+  >}}
+
+## Fixed regressions
+
+[Rei](https://github.com/Its-Rei) found a 3D texture regression in the Wooden Kingdom of `Super Mario Odyssey` that was affecting both Vulkan and OpenGL, 
+so [Blinkhawk](https://github.com/FernandoS27) worked on it and found out that there was a bug in the texture cache. 
+Now it works as intended again, no more fog stripes on trees.
+
+{{< single-title-imgs
+    "As it should be. (Super Mario Odyssey)"
+    "./17.png"
+    "./18.png"
+  >}}
+
+[Rodrigo](https://github.com/ReinUsesLisp) fixed a regression found on `Puyo Puyo Tetris` while running in OpenGL. 
+With it in place, character portraits in game have their proper white border rendered.
+
+{{< single-title-imgs
+    "Better, right? (Puyo Puyo Tetris)"
+    "./01.png"
+    "./02.png"
+  >}}
+
+[Rodrigo](https://github.com/ReinUsesLisp) also fixed a regression in `Kirby Star Allies`, restoring shading and life to this cute game. 
+Kirby deserves a colorful world for his adventures!
+
+{{< single-title-imgs
+    "Poyo! (Kirby Star Allies)"
+    "./25.png"
+    "./26.png"
+  >}}
+
+Yuzu can not only run commercial games, it can run homebrew programs too. [bunnei](https://github.com/bunnei) fixed a regression 
+on the new memory manager I talk about previously, restoring functionality to some homebrew games/apps that crashed.
```

Comment:
I think this can be moved to where the VMM was mentioned to make the article flow better

Reply:
Could be, I'm still contemplating if regressions should be its own category.

Reply:
I don't think regressions needs to be its own category. But this blurb should definitely be added at the end of this paragraph:

```[bunnei](https://github.com/bunnei) rewrote the Virtual Memory Manager (known internally as `Project Epimetheus`), drastically reducing RAM use by games, and helping `Project Prometheus` (multicore support) become a reality.```

With my modification in previous comments, and this move, the paragraph should look like this:

```[bunnei](https://github.com/bunnei) rewrote the Virtual Memory Manager (known internally as `Project Epimetheus`), which drastically reduced the amount of RAM games use, and helped Multicore support (also known as `Project Prometheus`) become a reality. This rewrite also resolved a crashing issue in some homebrew games/apps.```

Reply:
The VMM rewrite introduced that bug, that's why it is listed as regression fixes.